src/SourceBuild/README.md

```diff
@@ -74,4 +58,5 @@ There is a method to create a series of patches based on a range of Git commits,
 but this is not usually useful for 6.0 main development. It is used in servicing
 to "freshen up" the sequence of patches (resolve conflicts) all at once.
 
-Note: Tarballs have already applied patches to the source code.
+Note: The VMR has all of the `src/SourceBuild/patches` applied. This is done as part of the
```

Comment:
I am not 100% sure it's semantically correct, but I got used to doing this for notes in md:
```suggestion
> Note: The VMR has all of the `src/SourceBuild/patches` applied. This is done as part of the
```

(it gives it this nice visual clue)